### DISCUSSION TITLE
One player.....two speakers !!!

### DISCUSSION THREAD

**Discussion by Albo0n** [music-assistant/support#1798] [Feature requests and ideas]
Hello, I've a yamaha musiccast player. 
In HA musiccast create 1 media_player and 2 switchs (speaker A & B)
So I can't easely select A, B or A+B speakers in MA

**Comment by d3frost** [comment]
"In HA musiccast create 1 media_player and 2 switchs (speaker A & B"
I also have Musiccast Speakers ( 1 Soundbar and 4 Speakers ) But have no idea how you manage this with a Switch in HA ? Can you explain this for me ?

**Comment by Albo0n** [comment]
I ve 2 stereo output in my Yamaha amplifier. And the musiccast integration of Home Assistant create 2 Switch, for A and B output. It depens of your Yamaha product.

**Reply by d3frost** [reply]
Ah, okay ....... My Bar and the Rest of the Speakers only shown as Media Players ! 
Thank you !

**Reply by Albo0n** [reply]
![Screenshot_20240118_185215](https://github.com/music-assistant/support/assets/148807067/d23ba7f5-0b65-4e18-b877-348a72933087)

**Reply by OzGav** [reply]
Yes you cant do that at the moment as HA players are not visible to MA. In a future release this is planned however.

**Reply by OzGav** [reply]
Going through the older requests. Does this work now?

**Comment by OzGav** [comment]
MusicCast has been added to 2.6.0b6
